carma_wm/test/CARMAWorldModelTest.cpp

```diff
@@ -1005,4 +1005,18 @@ TEST(CARMAWorldModelTest, getLaneletsFromPoint)
   ASSERT_EQ(underlyings.back().id(), 1200);
 }
 
+TEST(CARMAWorldModelTest, setConfigSpeedLimitTest)
+{
+  CARMAWorldModel cmw;
+
+  bool flag = false;
+  double cL = 24.0;
+  ///// Test without user defined config limit
+  cmw.setConfigSpeedLimit(cL);
+
+  ASSERT_FALSE(flag);
```

Comment:
Nothing is actually being tested here. Since this method is called in other unit tests, I think this file is probably unnecessary. 

If this test was required to generate coverage than that makes me